### ISSUE TITLE
Change the background color of Boom menu Button

### ISSUE THREAD

**Issue by dimplepatel23** [Nightonke/BoomMenu#179]
I have written the code like this
for (int i = 0; i < bmb.getPiecePlaceEnum().pieceNumber(); i++) {
    HamButton.Builder builder = new HamButton.Builder()
            .normalImageRes(R.drawable.butterfly)
            .normalTextRes("Butter Doesn't fly!")
            .normalColor(R.color.colorPrimary)
            .subNormalTextRes("Little butter Doesn't fly, either!");
    bmb.addBuilder(builder);
}

The problem is it is not showing the exact color which i set in color.xml file 
Even the padding efffect is not applied to image in boom menu

**Comment by badarshahzad** [comment]
Yes I also find this bug. I tried to set the menus background color. I tried this color #F5A623 it doesn't work but this color code work #343d43.

```
 SimpleCircleButton.Builder b1 = new SimpleCircleButton.Builder();
            b1.normalImageRes(imageResources[i]);
            bmb.addBuilder(b1);

```

**Comment by imranali07** [comment]
Can I change background transparent in boom menu button ?

**Comment by Gerrix90** [comment]
> I have written the code like this
> for (int i = 0; i < bmb.getPiecePlaceEnum().pieceNumber(); i++) {
> HamButton.Builder builder = new HamButton.Builder()
> .normalImageRes(R.drawable.butterfly)
> .normalTextRes("Butter Doesn't fly!")
> .normalColor(R.color.colorPrimary)
> .subNormalTextRes("Little butter Doesn't fly, either!");
> bmb.addBuilder(builder);
> }
> 
> The problem is it is not showing the exact color which i set in color.xml file
> Even the padding efffect is not applied to image in boom menu

This is how you change color

        for (int i = 0; i < bmb.getPiecePlaceEnum().pieceNumber(); i++){
            TextInsideCircleButton.Builder builder = new TextInsideCircleButton.Builder()
                    .normalColor(getResources().getColor(R.color.colorAccent))
                    .normalImageRes(R.drawable.butterfly)
                    .normalText("Butter Doesn't fly!");
            bmb.addBuilder(builder);
        }

**Comment by srijanbiswas3** [comment]
> I have written the code like this
> for (int i = 0; i < bmb.getPiecePlaceEnum().pieceNumber(); i++) {
> HamButton.Builder builder = new HamButton.Builder()
> .normalImageRes(R.drawable.butterfly)
> .normalTextRes("Butter Doesn't fly!")
> .normalColor(R.color.colorPrimary)
> .subNormalTextRes("Little butter Doesn't fly, either!");
> bmb.addBuilder(builder);
> }
> 
> The problem is it is not showing the exact color which i set in color.xml file
> Even the padding efffect is not applied to image in boom menu


### you can set it by using rbg of the same color

```
HamButton.Builder builder = new HamButton.Builder()
.normalColor(Color.rgb(	114, 188, 212));

```
